garrulous-gobies/Bangazon-sprint-2
# add product form

## Link to Ticket
Closes # [https://github.com/garrulous-gobies/Bangazon-sprint-2/issues/1](issue #1 )

## Description of Proposed Changes
Adds form validation with custom error messages. Will not allow negative numbers for either price or quantity. Quantity must be at least 1. Price must be at least $.02.

#### Changes to the product model: price and quantity min value validators


## Steps to Test

```
git fetch --all
git checkout nl-sell-product
```
-make sure you have a product type with an id of 1 in your db
-test minimum field values for price and quantity. 


## Impacted Areas in Application

List general components of the application that this PR will affect:

* models
*forms

## Mentions @username

Tag users that need to review this code
